install-cockroachdb.md

```diff
@@ -22,31 +22,18 @@ $(document).ready(function(){
 });
 </script>
 
-<!--<script>
-$(document).ready(function(){
-$(".osbutton").click(function(){ $(".install").hide();});
-    $("#mac").click(function(){
-        $("#macinstall").show();
-    });
-    $("#linux").click(function(){
-        $("#linuxinstall").show();
-    });
-    $("#windows").click(function(){
-        $("#windowsinstall").show();
-    });
-});
-</script>-->
-
-<button id="mac">Mac</button>
+<button id="mac">Mac OS X</button>
 <button id="linux">Linux</button>
 <button id="windows">Windows</button>
 
+<!-- Install instructions for Mac OS X -->
 <div id=macinstall>
-<p>There are currently two ways to install CockroachDB locally on OSX:</p>
+<p>There are three ways to install CockroachDB locally on Mac OS X:</p>
```

Comment:
Filed https://github.com/cockroachdb/cockroach/issues/3959, but then @tamird pointed out that homebrew requires Xcode, which gets you a C++ compiler and git. So as long as our brew recipe gets Go, looks like we're covering all the dependencies.